app/controllers/categories_controller.rb

```diff
@@ -14,45 +15,33 @@ def show; end
   private
 
   def init_values
-    @active_more_btn = false
-    @page_number = 1
+    @present = CategoriesPresenter.new(params: params)
     load_books_with_category(params) if params[:id]
-    load_books_without_category(params) unless params[:id]
-    show_more_button?(params)
+    load_books_without_category unless params[:id]
+    @active_more_btn = show_more_button?(params)
```

Comment:
Is it possible to move `active_more_btn` to presenter?

Reply:
It's possible, but first I have to load all the books
![2018-08-14 23 44 29](https://user-images.githubusercontent.com/9571192/44117677-a5b56714-a01c-11e8-95d4-53fc031910f3.png)